src/main/java/io/supertokens/webserver/api/thirdparty/getUsersByEmail/GetUsersByEmailAPI.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Why is this file inside `api/thirdparty/getUsersByEmail/..` as opposed to being in `api/thirdparty/..`.

Please follow how other APIs are made. It just adds a lot of confusion.